package.json

```diff
@@ -18,13 +18,14 @@
   "dependencies": {
     "@prisma/client": "^4.15.0",
     "@types/cors": "^2.8.13",
-    "@types/node": "^20.3.0",
+    "@types/node": "^15.14.1",
```

Comment:
Why do we need degrade the version?

Reply:
Necessary for Prisma client to work. @Adityagupta1625 suggested this solution.